.gitpod.yml

```diff
@@ -6,19 +6,40 @@ github:
     branches: true
     # enable for pull requests coming from this repo (defaults to true)
     pullRequests: true
+    # enable for pull requests coming from forks (defaults to false)
+    pullRequestsFromForks: true
     # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
     addComment: true
     # configure whether Gitpod registers itself as a status check to pull requests
     addCheck: false
 
 tasks:
   - name: prebuild
-    init: ./scripts/start.sh && ./scripts/stop.sh && gp sync-done prebuild
-    command: curl -L --http1.1 https://cnfl.io/ccloud-cli | sudo sh -s -- -b /usr/local/bin; exit
+    init: |
+      cd .. && git clone https://github.com/confluentinc/jmx-monitoring-stacks && cd cp-demo/. && \
+        ./scripts/start.sh && ./scripts/stop.sh && \
```

Comment:
Something I'm planning to do anyway is to replace `./scripts/start.sh && ./scripts/stop.sh` with simply `CLEAN=true ./scripts/start.sh`. This makes a better experience if there is no prebuild enabled. There's no reason to require stop.sh. I think it's fine to add it to this PR because I don't want you to have to rebase and fix the conflict.